.gitignore

```diff
@@ -4,3 +4,4 @@
 /vendor/
 /coverage/
 /.phpunit.result.cache
+/.idea
```

Comment:
This should be placed in your global gitignore.

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Didn't know this existed. Thanks, will use this from now on